browser/src/control/jsdialog/Widget.Combobox.js

```diff
@@ -317,8 +317,11 @@ JSDialog.combobox = function (parentContainer, data, builder) {
 
 	container.onSelect = function (pos) {
 		resetSelection();
-		if (pos >= 0 && entries[pos])
+		if (pos >= 0 && entries[pos]) {
 			entries[pos].selected = true;
+			if (entries[pos].text)
+				this.onSetText(entries[pos].text);
```

Comment:
I would expect this in separate commit with description what it does, how to reproduce bug. what does it fix?

Reply:
So ideally i don't like this, it feels hacky but your opinion may help here. 


so context toolbar is treated as notebook bar. When I change i.e Font, core would send back the update for the sidebar but notebook bar is already updated via user selection in js and core doesn't send any update for fontnamecombobox because its redundant. 

so in situations when you updated font or font size from notebookbar or context toolbar the text is not updated vice versa. These updates are linked with window id, in this case I am using -2 which is used for the notebookbar. 


is there any better way to do it? i could change and force core to send update in all situation but thought that is just waste of bandwidth.